src/commands/remember/remember-chunk.js

```diff
@@ -20,25 +61,38 @@ module.exports = {
         },
         {
             name: 'speed',
-            description: 'Speed of the Search. Lower value is more accurate but slower.',
+            description: 'Speed of the Search. Lower value is more accurate but slower. Range 25-100 inclusive',
             type: ApplicationCommandOptionType.Number,
         },
+        {
+            name: 'exclude-bot-messages',
+            description: `If bot messages should be excluded in the message collection. Default is ${defaultExcludeBotMessages}`,
+            type: ApplicationCommandOptionType.Boolean
+        }
     ],
 
     //logic, 
     callback: async(client, interaction) =>{
         try{
             await interaction.deferReply();
 
-            //get the id    
-            const value = interaction.options.get('minutes').value;
+            //get the hours   
+            const valueHours = interaction.options.get('hours').value;
+
+            //get the minutes   
+            const valueMinutes = interaction.options.get('minutes').value;
+
+            //bot messages
+            const excludeBotMessages = interaction.options.getBoolean('exclude-bot-messages') ?? defaultExcludeBotMessages;
 
             //validate accuracy
-            let accuracy = interaction.options.getNumber("speed") ?? 100;
-            if(accuracy < 25) accuracy = 25; 
-            if(accuracy > 100) accuracy = 100;
+            let accuracy = interaction.options.getNumber("speed") ?? 50;
+            accuracy = Math.max (accuracy, 25); 
+            accuracy = Math.min(accuracy, 100);
```

Comment:
I wasn't paying attention in my initial review :/ No need to change anything here, just a tip. This process is usually called "clamp", so I'll sometimes do something like this:
```js
const clamp = (min, max, num) => Math.max(min, Math.min(num, max));
let accuracy = clamp(25, 100, interaction.options.getNumber("speed") ?? 50);
```
Benefit is that it's self-documenting, and (I think at least) helps your mind abstract over minute details. Plus you're actually clamping several times in this file, so I might recommend making clamp a global in this file and just using throughout.